googlemock/test/gmock-more-actions_test.cc

```diff
@@ -74,7 +74,7 @@ class NullaryFunctor {
   int operator()() { return 2; }
 };
 
-bool g_done = false;
+static bool g_done = false;
```

Comment:
Let's put the whole file in an anonymous namespace instead

Reply:
This file is already nested in a non-anonymous namespace. Refactoring the test to make it all be anonymous would be considerably more messy than making a single variable static within that namespace.

As a sidenote: the use of static vs anonymous namespace is inconsistently applied. Some code uses `static`; others use anonymous namespace.